pallets/gear/src/tests.rs

```diff
@@ -57,6 +58,9 @@ use utils::*;
 #[test]
 fn unstoppable_block_execution_works() {
     init_logger();
+
+    let minimal_weight = get_minimal_weight();
```

Comment:
Better to mention the module when calling functions:
```suggestion
    let minimal_weight = mock::get_minimal_weight();
```